python/tvm/relay/op/transform.py

```diff
@@ -0,0 +1,46 @@
+"""Transform tensor operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+def flatten(data):
```

Comment:
The `flatten` here is not consistent with numpy's flatten (https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.flatten.html). Why not use names like `batch_flatten`?

Reply:
+1 for batch_flatten